uPortal-webapp/src/main/webapp/media/skins/respondr/common/less/navigation.less

```diff
@@ -526,20 +526,20 @@
                                         color: @grayscale10;
                                         margin: 0 auto;
                                     }
-                                    p.fl-inlineEdit-editModeInstruction {
+                                    p {
                                         height: 0;
                                         overflow: hidden;
                                         width: 0;
                                     }
                                 }
 
                                 &.useflyout {
-                                    a.portal-navigation-link, div.fl-inlineEditContainer {
+                                    a.portal-navigation-link, div {
                                         padding: 1px 5px;
                                         width: auto;
                                     }
 
-                                    div.fl-inlineEditContainer {
+                                    div {
```

Comment:
worrisome.

I'm thinking all the elements (and LESS selectors) that lost a class should be given a new class that starts with `up-`.